product_brand/readme/ROADMAP.rst

```diff
@@ -0,0 +1,3 @@
+* Add a field with brands associated to a Customer or Supplier on
+  the Customers/Suppliers Form View.
+* Fix smart button alignment in brand form view
```

Comment:
Is this still happening?

Reply:
Well, now that I look at it, I don't think so :smiley: 
I will remove it :+1: